plunger/model.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of Plunger.
+#
+# Copyright (C) 2017 reverendus
```

Comment:
Old copyright

Reply:
I didn't change anything, so didn't see fit modifying the copyright just for pulling the code into a new file.  Originally I had a separated etherscan into another class.  Since that was removed, I think I'll eliminate this file and put `Transaction` back where it was.

Reply:
Done.